### DISCUSSION TITLE
Backend partial override

### DISCUSSION THREAD

**Discussion by Tony-S** [wintercms/winter#311] [Q&A] [answered]
I'm trying to override backend/behaviors/importexportcontroller/partials/_import_db_columns.htm
So in my plugin file I have:
```
public function boot()
    {
        ImportExportController::extend(function($controller){
            $controller->addViewPath(plugins_path() . '/technigrafix/catalogue/partials');
            Log::info(print_r($controller->getViewPaths(),1));
        });
    }
```
The log is just to check things are working and I get:
```
Array
(
    [0] => /home/xxxxx/public_html/xxxx/plugins/technigrafix/catalogue/partials
    [1] => /home/xxxxx/public_html/xxxx/modules/backend/behaviors/importexportcontroller/partials
)
```
so that looks to have worked. The modified partial is in plugins/technigrafix/catalogue/partials but it doesn't get picked up. If I modify the core partial then that works. Is there something I'm missing?
Thanks

**Comment by LukeTowers** [comment, marked as answer]
It's not documented yet but you can prefix a given ImportExport partial with `_import_export` and put it in your controller's folder and it will override the default partials. I.e. if you put `_import_export_import_db_columns.htm` in `myauthor/myplugin/controllers/catalogue/` then it would load your local override instead. 

Apart from that, the only other thing I can think of is if you are on a case sensitive file system and your path doesn't match the actual path on disk 100% then it could cause issues, or if you have permission issues on the folders or files.
